klog.go

```diff
@@ -412,7 +412,7 @@ func InitFlags(flagset *flag.FlagSet) {
 	flagset.StringVar(&logging.logFile, "log_file", "", "If non-empty, use this log file")
 	flagset.BoolVar(&logging.toStderr, "logtostderr", false, "log to standard error instead of files")
 	flagset.BoolVar(&logging.alsoToStderr, "alsologtostderr", false, "log to standard error as well as files")
-	flagset.Var(&logging.verbosity, "v", "log level for V logs")
+	flagset.Var(&logging.verbosity, "v", "log level verbosity followed by an integer representing the log level")
```

Comment:
i'd say the original is kind of OK.

if we are to change it it can be just:
> log verbosity level

Reply:
Imho We should give information that loghvel are integers. Normally we use words for log levels, this is not clear with the original one

Reply:
this is a bit confusingly worded right now.  I'd say `integer verbosity for V logs` or something similar.  `log level verbosity followed by` makes it seem like it takes two arguments.

Reply:
seems pretty reasonable.

Reply:
thx! adressed at https://github.com/kubernetes/klog/pull/32/files#diff-fb778b051a785bbb2eea40d6a00a3a21R415